### DISCUSSION TITLE
Bling my thing!

### DISCUSSION THREAD

**Discussion by sgenoud** [sgenoud/replicad#3] [Show and tell]
A new project I have started using replicad: https://blingmything.sgenoud.com/

<img width="1121" alt="Capture d’écran 2022-03-18 à 16 32 01" src="https://user-images.githubusercontent.com/263325/159033713-49a54227-706c-4b01-9c56-820262385550.png">

This is for a now a simple editor that allow you to write on faces of a (STEP) model.

**Comment by lf94** [comment]
Could you instead opt for any arbitrary SVG?

**Reply by sgenoud** [reply]
I would like to support SVGs as well - but arbitrary ones are difficult. If you feel like tackling SVG imports in replicad I would love to include it in pimpmything!

**Reply by lf94** [reply]
I have a lot of experience now with SVG having done it for Curv: https://github.com/curv3d/svg2curv the code is probably pretty adaptable to replicad! I'll look into it :)

**Reply by sgenoud** [reply]
SVG import achieved! I have not put the code in replicad - this is a best effort import (and some code has some edge cases that need to be taken care of). But you can import (most) SVGs now: 

<img width="672" alt="Capture d’écran 2022-03-31 à 15 32 20" src="https://user-images.githubusercontent.com/263325/161066957-f8e5448e-c56e-4d75-9738-fa9b990d21e9.png">

**Reply by lf94** [reply]
Niiice! Was my project helpful at all or nah?

**Reply by sgenoud** [reply]
The difficulty came from merging different path - I don't think that your implementation handles it (yet). That said it motivated me to just dig in it!
